src/server/game/Spells/Auras/SpellAuras.cpp

```diff
@@ -191,7 +191,7 @@ void AuraApplication::BuildUpdatePacket(WorldPackets::Spells::AuraInfo& auraInfo
 
     // send stack amount for aura which could be stacked (never 0 - causes incorrect display) or charges
     // stack amount has priority over charges (checked on retail with spell 50262)
-    auraData.Applications = aura->GetSpellInfo()->StackAmount ? aura->GetStackAmount() : aura->GetCharges();
+    auraData.Applications = aura->GetMaxStackAmount() ? aura->GetStackAmount() : aura->GetCharges();
```

Comment:
I would rather not do unneccessary caster lookup to calculate max stacks since we are not interested in that value here, just to know if aura uses stacks or charges

Reply:
What if the spell has 0 on both stacks and charges, but can be "moded" (is that a verb?) to have more stacks? or is that not a possible scenario?